variable/technology/technologies.yaml

```diff
@@ -460,3 +531,326 @@ Investment|Energy Supply|Liquids|Oil:
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Gas:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Hydro:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Nuclear:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Oil:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Wind:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Solar:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Ramp Up|Electricity:
+   description: Upward ramp limit of the power generation unit
+   unit: GW/h
+
+Ramp Up|Electricity|Coal:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Up|Electricity|Gas:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Up|Electricity|Nuclear:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Up|Electricity|Oil:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Down|Electricity:
+   description: Downward ramp limit of the power generation unit
+   unit: GW/h
+
+Ramp Down|Electricity|Coal:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Down|Electricity|Gas:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Down|Electricity|Nuclear:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Down|Electricity|Oil:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Reactance|Electricity|Transmission|Line:
+   description: Reactance. Lines need to have a reactance different from 0 in the case of a DC cable or a transportation modeling for an AC cable
+   unit: p.u.
+
+Security Factor|Electricity|Transmission|Line:
+   description: Security factor to consider approximately N-1 contingencies. NTC = TTC x SecurityFactor
+   unit: p.u.
```

Comment:
what is `p.u.`? If this is just a multiplicative factor, just write `unit:` (i.e., no unit).